docs/coding.rst

```diff
@@ -360,3 +360,432 @@ Bedrock includes an `.editorconfig` file in the root directory that you can
+
+    Default: N/A
+
+    Example: ``link_cta=ftl('about-read-the-manifesto')``
+
+- link_url
+    **Required**. String or url helper function provides href value for cta link. 
+
+    Default: N/A
+
+    Example: ``link_url=url('mozorg.about.manifesto')``   
+
+- image_url 
+    **Required**. Path to image location.
+
+    Default: N/A
+
+    Example: ``image_url='img/home/2018/billboard-healthy-internet.png'`` 
+
+- include_highres_image 
+    Boolean that determines whether the image can also appear in high res.
+
+    Default: False
+
+    Example: ``include_highres_image=True``   
+
+- reverse  
+    Uses default layout: mzp-l-billboard-rightReverse will switch to billboard (text) left.	
+
+    Default: False
+
+    Example: ``reverse=True`` 
+
+- heading_level 
+    Number indicating heading level for title text. Should be based on semantic meaning, not presentational styling.
+
+    Default: 2	
+
+    Example: ``heading_level=1`` 
+
+
+Feature Card
+^^^^^^^^^^^^
+
+- title
+    String indicating heading text (usually a translation id wrapped in ftl function).
+
+    Default: None
+
+    Example: ``title=ftl('firefox-home-firefox-browser')``
+
+- ga_title 
+    String used as an identifying name on a link for google analytics. Only used if link_url and link_cta are provided as well.
+
+    Default: None
+
+    Example: ``ga_title='Firefox Windows'``
+
+- image_url
+    Path to image location.
+
+    Default: N/A
+
+    Example: ``image_url=’img/firefox/accounts/trailhead/value-respect.jpg’`` 
+
+- class
+    String adding class(es) to the section tag.
+
+    Default: None
+
+    Example: ``class=’mzp-l-card-feature-left-half t-mozvpn’``   
+
+- link_url
+    String or url helper function provides href value for cta link. Only used if link_cta is provided as well.
+
+    Default: None
+
+    Example: ``link_url=url('firefox.privacy.index')`` 
+
+- link_cta
+    String indicating link text (usually a translation id wrapped in an ftl function). Only used if link_url is provided as well.
+
+    Default: None
+
+    Example: ``link_cta=ftl('ui-learn-more')``
+
+- include_highres_image
+    Boolean that determines whether the image can also appear in high res.
+
+    Default: False
+
+    Example: ``include_highres_image=True``   
+
+- l10n_image 
+    Boolean to indicate if image has translatable text.
+
+    Default: False
+
+    Example: ``l10n_image=True`` 
+
+- aspect_ratio
+    aspect_ratio 	String with an mzp class name indicating desired aspect ratio (adds class to section tag).
+
+    Default: False
+
+    Example: ``aspect_ratio='mzp-has-aspect-3-2'``   
+
+- heading_level 
+    Number indicating heading level for title text. Should be based on semantic meaning, not presentational styling.
+
+    Default: 2
+
+    Example: ``heading_level=3`` 
+
+- media_after 	
+    Boolean to determine if image appears before or after text when stacked on mobile size screens.
+
+    Default: False
+
+    Example: ``media_after=True`` 
+
+
+Card
```

Comment:
I forgot to add on the [Google Doc](https://docs.google.com/document/d/1Ux8iu8e7lbopwjogQywbCtbKAYuQoZW4Kkfj85WNL18/edit) another parameter for the Card component, called `youtube_id`. I added it to the bottom of the doc if you could add it :)

Reply:
I've already told you that this Google doc is not accessible to me so just send the actual parameter, please.

Reply:
Ah, thanks for the reminder. Here ya go:

<div dir="ltr" style="margin-left:-38.25pt;" align="left" id="docs-internal-guid-5846eb71-7fff-dcb8-4986-b5fc8d591737">

Name | Explanation | Required? | Default | Example
-- | -- | -- | -- | --
youtube_id | String indicating the Youtube ID found at the end of a Youtube video URL. Used when we are embedding a video to the card rather than an image | No | N/A | youtube_id=’aHpCLDQ_2ns`

</div>

Reply:
Done @reemhamz